drivers/bluetooth/hci/Kconfig

```diff
@@ -16,7 +16,7 @@ config BT_H4
 	select UART_INTERRUPT_DRIVEN
 	select BT_UART
 	select BT_RECV_IS_RX_THREAD
-	depends on SERIAL
+	depends on SERIAL && SERIAL_SUPPORT_INTERRUPT
```

Comment:
Since `SERIAL_SUPPORT_INTERRUPT` implicitly depends on `SERIAL` wouldn't it be enough to depend on just `SERIAL_SUPPORT_INTERRUPT`?

Reply:
changed.